Core/KSP.cs

```diff
@@ -118,6 +122,12 @@ public void Dispose()
                 Cache.Dispose();
         }
 
+
+        ///public void Dispose()
+        ///{
+        ///    this.RegistryManager.Dispose();
```

Comment:
If this remains commented out, an additional comment explaining why it was commented out or what it did would be helpful.